plugins/inputs/bond/bond_test.go

```diff
@@ -0,0 +1,77 @@
+package bond
+
+import (
+	"testing"
+
+	"github.com/influxdata/telegraf/testutil"
+)
+
+var sampleTest802 = `
+Ethernet Channel Bonding Driver: v3.5.0 (November 4, 2008)
+
+Bonding Mode: IEEE 802.3ad Dynamic link aggregation
+Transmit Hash Policy: layer2 (0)
+MII Status: up
+MII Polling Interval (ms): 100
+Up Delay (ms): 0
+Down Delay (ms): 0
+
+802.3ad info
+LACP rate: fast
+Aggregator selection policy (ad_select): stable
+bond bond0 has no active aggregator
+
+Slave Interface: eth1
+MII Status: up
+Link Failure Count: 0
+Permanent HW addr: 00:0c:29:f5:b7:11
+Aggregator ID: N/A
+
+Slave Interface: eth2
+MII Status: up
+Link Failure Count: 3
+Permanent HW addr: 00:0c:29:f5:b7:1b
+Aggregator ID: N/A
+`
+
+var sampleTestAB = `
+Ethernet Channel Bonding Driver: v3.6.0 (September 26, 2009)
+
+Bonding Mode: fault-tolerance (active-backup)
+Primary Slave: eth2 (primary_reselect always)
+Currently Active Slave: eth2
+MII Status: up
+MII Polling Interval (ms): 100
+Up Delay (ms): 0
+Down Delay (ms): 0
+
+Slave Interface: eth3
+MII Status: down
```

Comment:
It's also possible for both of the interfaces to be up, right?  Maybe we should gather the active slave on the bond master?

Reply:
Sure, I've added active_slave field. Thanks for the note.